src/erlcloud_ssm.erl

```diff
@@ -0,0 +1,601 @@
+%% @doc
+%% An Erlang interface to AWS Systems Manager (SSM).
+%%
+%% Output is in the form of `{ok, Value}' or `{error, Reason}'. The
+%% format of `Value' is controlled by the `out' option, which defaults
+%% to `json'. The possible values are:
+%%
+%% * `json' - The output from Systems Manager as processed by `jsx:decode'
+%% but with no further manipulation.
+%%
+%% * `record' - A record containing all the information from the
+%% Systems Manager response except field types.
+%%
+%% * `map' - A map containing all the information from the
+%% Systems Manager response except field types.
```

Comment:
what does this mean? I think the output must be the same as `json` but map form?